ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/tools/TestExecutorTest.java

```diff
@@ -414,9 +415,9 @@ public void shouldPassOnExpectedOutput() {
     final ProducerRecord<byte[], byte[]> rec1 = producerRecord(sinkTopic, 123456789L, "k2", "v2");
     when(kafkaService.readRecords(SINK_TOPIC_NAME)).thenReturn(ImmutableList.of(rec0, rec1));
 
-    final Record expected_0 = new Record(SINK_TOPIC_NAME, "k1", "v1", TextNode.valueOf("v1"),
+    final Record expected_0 = new Record(SINK_TOPIC_NAME, "k1", JsonNodeFactory.instance.textNode("k1"), "v1", TextNode.valueOf("v1"),
```

Comment:
Is this different from `TextNode.valueOf("k1")`? Curious about the inconsistency between the key and value nodes.

Reply:
🙄 to be honest, I've just always used `JsonNodeFactory` to create the nodes (I didn't even notice that they were using `TextNode` here). I don't think there's a difference:
```java
    // From JsonNodeFactory
    public TextNode textNode(String text) { return TextNode.valueOf(text); }
```